### PR TITLE
Vim plugins n grammar

### PR DESCRIPTION
**Pull request by edwtjo**
###### Motivation for this change

Adds a grammar checker to vim + agda-vim + some updates.

To use the grammar checker set `let g:grammarous#jar_dir="${languagetool}/share/"` in vimrc and also add "vim-grammarous" to your plugin set.
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

@jagajaga

### CODE REVIEW

File: pkgs/tools/text/languagetool/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl, writeScriptBin, unzip, jdk, ... }:
```
Comment (groxxda) [inline pkgs/tools/text/languagetool/default.nix]: Please remove the ellipsis (`...`). And `writeBinScript` is not used, right?

File: pkgs/tools/text/languagetool/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    EXE
+    chmod +x $out/bin/$lt
+    done
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://languagetool.org";
+    license = licenses.lgpg21Plus;
+    maintainers = with maintainers; [ edwtjo ];
+    descrption = "A proofreading program for English, French German, Polish, ad more.";
```
Comment (groxxda) [inline pkgs/tools/text/languagetool/default.nix]: Typos :)

Comment (jagajaga) [inline pkgs/tools/text/languagetool/default.nix]: Also dot at the end

**Comment by edwtjo** [comment]
Thanks guys!

**Comment by jagajaga** [comment]
@edwtjo why did you close it?

**Comment by garbas** [comment]
i think "comment" and "close and comment" buttons are to close :smile:

**Comment by edwtjo** [comment]
@jagajaga @garbas heh no I cherry-picked the commits before closing :) I should've written "closed with <rev> <rev>"

**Pull request merged**